src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaConstants.java

```diff
@@ -8,10 +8,6 @@
 
 	String DEFAULT_SCOPE = "goose"; // TODO replace this by the config option "storage-namespace" value
 
-	int CONTROL_API_TIMEOUT_MILLIS = 30_000;
-
-	int CLOSE_TIMEOUT_MILLIS = 30_000;
```

Comment:
Where did these parameters go?

Reply:
these 2 options moved into the single configuration option `storage-driver-control-timeoutMillis`